### PR TITLE
Add Official AzuraCast One-Click App Manifest

### PR DESCRIPTION
**Pull request by BusterNeece**
Hello! I'm the creator and lead maintainer of the [AzuraCast](https://github.com/AzuraCast/AzuraCast) web radio management suite. Our application is primarily deployed via Docker, so CapRover makes excellent sense as a deployment method for our users. I'm hoping we can get added to the official app list!

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.
- [x] I've executed the checks if necessary by running `npm ci && npm run validate_apps && npm run formatter`
- [x] I will take responsibility addressing any issues that arises as a result of this PR (maintaining this app).

### CODE REVIEW

File: public/v4/apps/azuracast.yml [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+            - '8466:8466'
+            - '8470:8470'
+            - '8475:8475'
+            - '8476:8476'
+            - '8480:8480'
+            - '8485:8485'
+            - '8486:8486'
+            - '8490:8490'
+            - '8495:8495'
+            - '8496:8496'
```
Comment (githubsaturn) [inline public/v4/apps/azuracast.yml:158]: Are these ports all required? If so, I suggest adding a warning note in `instructions.start` for the user to make sure all these ports are open and they are not used by any other applications. If they are already in-use the container fail to start up.

**Comment by githubsaturn** [comment]
Thanks for the PR! Just a minor comment. Also the build is failing. Probably some formatting issues.

**Comment by BusterNeece** [comment]
@githubsaturn Thank you! Formatting fixes are applied and I've added a description about the heavy port clearance.

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**